klippy/extras/tmc2209.py

```diff
@@ -0,0 +1,92 @@
+# TMC2209 configuration
+#
+# Copyright (C) 2019  Stephan Oelze stephan.oelze@gmail.com
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+import tmc2208, tmc2130, tmc, tmc_uart
+TMC_FREQUENCY=12000000.
+
+Registers = dict(tmc2208.Registers)
+
+Registers.update({
+    "TCOOLTHRS": 0x14,
+    "COOLCONF": 0x42,
+    "SGTHRS": 0x40
+})
+
+ReadRegisters = tmc2208.ReadRegisters
```

Comment:
FYI, the tmc2209 has additional register that can be read (eg, SG_RESULT).

Reply:
i will add that, thanks